src/datetime/DaysView.js

```diff
@@ -76,7 +75,9 @@ export default class DaysView extends React.Component {
 		));
 	}
 
-	renderDay( date, startOfMonth, endOfMonth ) {
+	renderDay( date ) {
+		const startOfMonth = this.props.viewDate.clone().startOf('month');
```

Comment:
Parameter simplification is great, but in this case I think we can take `startOfMonth` and `endOfMonth` out to `renderDays`, so we clone the dates just once, instead of once for every single day and save some processing.

Reply:
What do you think about memoization? We can use a library like [`memoize-one`](https://github.com/alexreardon/memoize-one) to make these memoized methods.

Reply:
I love memoize-one! I use it in all my apps. I would resist to add it as a dependency here though. At least until we clean the code and see that is really needed for some performace issue.

Reply:
Done.